.readme-partials.yaml

```diff
@@ -0,0 +1 @@
+This library is meant to demonstrate authentication for sample purposes; it should not be used in production. To explore authentication options, see our [authentication library for Node](https://github.com/googleapis/google-auth-library-nodejs).
```

Comment:
I mean this is mostly true.  It shouldn't be used on the server-side, but it's entirely viable for a local CLI.

Reply:
Do you have suggestions on wording? For context, these are the expectations we're trying to set: https://github.com/googleapis/nodejs-local-auth/issues/7

Reply:
Rather than "should not be used in production", why don't we say "it should be treated as a starting point for building an application, and is not a general purpose solution".